corehq/blobs/migrate.py

```diff
@@ -649,6 +650,7 @@ def migrate(self, filename=None, reset=False, max_retry=2, chunk_size=100, limit
 
 
 MIGRATIONS = {m.slug: m for m in [
+    Migrator('domains', [Domain], CouchAttachmentMigrator),
```

Comment:
We normally write tests for these. See [test_migrate.py](https://github.com/dimagi/commcare-hq/blob/76b69907c78bb26b450c6fb4f3281defc3c1783a/corehq/blobs/tests/test_migrate.py#L164-L201)

Reply:
https://github.com/dimagi/commcare-hq/pull/21239/commits/24ed7a6f8d65f19e4070e8c4e9c3437fd8b1fa46